nixos/modules/installer/cd-dvd/installation-cd-minimal.nix

```diff
@@ -7,4 +7,7 @@
   imports =
     [ ./installation-cd-base.nix
     ];
+
+  # Enable cloud-init so the image can also be used for cloud providers
+  services.cloud-init.enable = true;
```

Comment:
This adds python + libraries to the iso. How much does this increase the ISO?

Reply:
Python 2.7 is already in the minimal image, so I'm hoping not much. I hope I can find out via ofBorg :)

Reply:
ouch, running `nix-shell -p cloud-init` on the minimal ISO turns into a 60MB download and 500MB unpacked :-/

Not sure if that is the best way to test though.